### DISCUSSION TITLE
beta feedback: not showing task list progress in board view

### DISCUSSION THREAD

**Discussion by newtmitch** [community/community#10437] [Projects and Issues]
In the new beta project capability, the cards on the board don't show task list progress like the older project capability did:

![image](https://user-images.githubusercontent.com/9440522/150647375-3c32cfc2-13d5-4f69-8154-3f6877ca4c98.png)

That's really handy to show progress for an individual item without having to dig into it.

**Comment by andrewhsu** [comment]
is this suggestion a duplicate of https://github.com/github/feedback/discussions/7575 ?
